client/my-sites/plans/main.jsx

```diff
@@ -36,9 +41,54 @@ const Plans = React.createClass( {
 		};
 	},
 
+	shouldDisplayJetpackPlansDisabled() {
+		const { selectedSite } = this.props;
+
+		return selectedSite && selectedSite.jetpack && ! isEnabled( 'manage/jetpack-plans' );
+	},
+
+	componentDidMount() {
+		const { context, selectedSite } = this.props,
+			analyticsPageTitle = 'Plans',
+			basePath = route.sectionify( context.path ),
+			analyticsBasePath = ( selectedSite ) ? basePath + '/:site' : basePath;
+
+		// FIXME: Auto-converted from the Flux setTitle action. Please use <DocumentHead> instead.
+		context.store.dispatch( setTitle( i18n.translate( 'Plans', { textOnly: true } ) ) );
+
+		if ( this.shouldDisplayJetpackPlansDisabled() ) {
+			analytics.pageView.record( basePath + '/jetpack/:site', analyticsPageTitle + ' > Jetpack Plans Not Available' );
+		} else {
+			analytics.tracks.recordEvent( 'calypso_plans_view' );
+			analytics.pageView.record( analyticsBasePath, analyticsPageTitle );
+		}
+
+		// Scroll to the top
+		if ( typeof window !== 'undefined' ) {
+			window.scrollTo( 0, 0 );
+		}
+	},
+
+	renderJetpackPlansDisabled() {
+		return (
+			<MainComponent>
+				<EmptyContentComponent
+					title={ i18n.translate( 'Plans are not available for Jetpack sites yet.' ) }
```

Comment:
Actually, can we remove this?  'manage/jetpack-plans' is enabled in all envs, and I'm not sure we'd ever turn it off.